embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/logging/InternalEmbraceLogger.kt

```diff
@@ -13,14 +12,14 @@ import java.util.concurrent.CopyOnWriteArrayList
 // perform as fast as possible.
 @Suppress("NOTHING_TO_INLINE")
 internal class InternalEmbraceLogger {
-    private val loggerActions = CopyOnWriteArrayList<LoggerAction>(listOf(AndroidLogger()))
+    private val loggerActions = mutableListOf<LoggerAction>(AndroidLoggingAction())
```

Comment:
This should stay as a `CopyOnWriteArrayList` as log functions can get called from multiple threads & a `LoggerAction` could get added at any time. Use of `mutableListOf` could result in concurrent modification exceptions